### ISSUE TITLE
react-apollo 2.3.0 includes all/most of the repository

### ISSUE THREAD

**Issue by arthens** [apollographql/react-apollo#2591]
I just upgraded from `react-apollo` 2.2.4 to 2.3.0 and my build started failing due to invalid syntax in apollo files. Upon further investigation I noticed that the package content is completely different.

2.2.4:

```
ls -a node_modules/react-apollo
.                               ApolloProvider.js.map           Query.js.map                    browser.js.map                  graphql.d.ts                    index.js                        parser.js                       queryRecycler.js.map            subscription-hoc.d.ts           test-utils.js                   withApollo.js.map
..                              LICENSE                         README.md                       component-utils.d.ts            graphql.js                      index.js.map                    parser.js.map                   react-apollo.browser.umd.js     subscription-hoc.js             test-utils.js.map
ApolloConsumer.d.ts             Mutation.d.ts                   Subscriptions.d.ts              component-utils.js              graphql.js.map                  mutation-hoc.d.ts               query-hoc.d.ts                  react-apollo.browser.umd.js.map subscription-hoc.js.map         types.d.ts
ApolloConsumer.js               Mutation.js                     Subscriptions.js                component-utils.js.map          hoc-utils.d.ts                  mutation-hoc.js                 query-hoc.js                    react-apollo.umd.js             test-links.d.ts                 types.js
ApolloConsumer.js.map           Mutation.js.map                 Subscriptions.js.map            getDataFromTree.d.ts            hoc-utils.js                    mutation-hoc.js.map             query-hoc.js.map                renderToStringWithData.d.ts     test-links.js                   types.js.map
ApolloProvider.d.ts             Query.d.ts                      browser.d.ts                    getDataFromTree.js              hoc-utils.js.map                package.json                    queryRecycler.d.ts              renderToStringWithData.js       test-links.js.map               withApollo.d.ts
ApolloProvider.js               Query.js                        browser.js                      getDataFromTree.js.map          index.d.ts                      parser.d.ts                     queryRecycler.js                renderToStringWithData.js.map   test-utils.d.ts                 withApollo.js
```

2.3.0:

```
ls -a node_modules/react-apollo
.                          .github                    Changelog.md               ROADMAP.md                 dist                       jest.cjs.config.js         lib                        rollup.config.js           test                       tslint.json
..                         .prettierignore            LICENSE                    coverage                   examples                   jest.preact.config.json    package.json               scripts                    test-examples.sh           typings
.circleci                  CONTRIBUTING.md            README.md                  dangerfile.ts              jest.browser.umd.config.js jest.server.umd.config.js  renovate.json              src                        tsconfig.json
```

Basically the latest release is shipping the whole repository instead of just shipping the compiled content in `lib`. Is this intentional? Is my webpack config meant to know to just look inside `lib` and ignore everything else?

**Version**

- react-apollo@2.3.0

**Comment by arthens** [comment]
Was this accidentally released? The changelog doesn't have any mention of `2.3.0`

**Comment by benjamn** [comment]
Version 2.3.0 was published incorrectly, though it was never promoted from `next` to `latest` on npm. These problems should be fixed now in version 2.3.1, and I just updated [`Changelog.md`](https://github.com/apollographql/react-apollo/blob/master/Changelog.md) to reflect the (intentional) changes.
